### DISCUSSION TITLE
loop property error

### DISCUSSION THREAD

**Discussion by lash0000** [nolimits4web/swiper#4477] [Q&A] [answered]
When i use a **loop** property in SwiperJS it gives me a error

Here's my config: 
![code](https://user-images.githubusercontent.com/38674978/115985140-024eb480-a5dd-11eb-943b-a2aaa90ec166.png)

The results: 
![loopissue](https://user-images.githubusercontent.com/38674978/115985145-0f6ba380-a5dd-11eb-8f02-a12aa2dfbe77.PNG)

**Comment by vltansky** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/5851280/115986075-462ec300-a5b7-11eb-9825-ee5751409243.png)

https://swiperjs.com/swiper-api

**Comment by lash0000** [comment]
Okay thank u
